app/factories/bulkrax/object_factory.rb

```diff
@@ -218,7 +221,7 @@ def update_attributes
       # a way that is compatible with how the factory needs them.
       def transform_attributes
         attributes.slice(*permitted_attributes)
-                  .merge(file_attributes)
+                  .merge(file_attributes(@update_files))
```

Comment:
i'd suggest using an accessor here. even if it is just `attr_accessor :update_files`, by avoiding using `@update_files` you gain a lot of flexibility by making everything a message (local variables and methods) and not using instance variables directly.

Reply:
I was already setting it as a attr_reader but then didn't use it! thanks for catching this rookie mistake!

Reply:
not rookie, just the kinda thing that gets left